src/QueryFactory/FindObjectsFromRawSqlQueryFactory.php

```diff
@@ -130,12 +132,46 @@ private function processParsedUnionQuery(array $parsedSql, ?string $sqlCount): a
 
         $generator = new PHPSQLCreator();
```

Comment:
Is it still necesarry ? could this still be used instead of the custom `buildUnion` ?

Reply:
yes the generator is used to create the processed sql count query just below the `buildUnion` call

`$processedSqlCount = $generator->create($countQuery);`

Reply:
In fact didn't notice, thanks --'